luna/featurevis/featurevis.py

```diff
@@ -2,106 +2,16 @@
-        first. Defaults to False.
-
-    Returns:
-        list: the modified image
-    """
-    if noise:
-        if channels_first:
-            img_noise = tf.random.uniform((1, 3, len(img[2]), len(img[3])),
-                                          dtype=tf.dtypes.float32)
-        else:
-            img_noise = tf.random.uniform((1, len(img[1]), len(img[2]), 3),
-                                          dtype=tf.dtypes.float32)
-        img_noise = (img_noise - 0.5) * 0.25 * ((100 - pctg) / 100)
-        img = img + img_noise
-        img = tf.clip_by_value(img, -1, 1)
-    return img
-
-
-def blur_image(img, blur, pctg):
-    """Gaussian blur the image to be modified.
-
-    Args:
-        img (list): the image to be blurred
-        blur (boolean): whether to blur the image
-        pctg (number): how much blur should be applied
-
-    Returns:
-        list: the blurred image
-    """
-    if blur:
-        img = gaussian_blur(img, sigma=0.001 + ((100-pctg) / 100) * 1)
-        img = tf.clip_by_value(img, -1, 1)
-    return img
-
-
-def rescale_image(img, scale, pctg):
-    """
-    Will rescale the current state of the image
-
-    :param img: the current state of the feature vis image
-    :param scale: true, if image should be randomly scaled
-    :param pctg: the amount of scaling in percentage
-    :return: the altered image
-    """
-    if scale:
-        scale_factor = tf.random.normal([1], 1, pctg)
-        img *= scale_factor[0]  # not working
-    return img
-
-
-# https://gist.github.com/blzq/c87d42f45a8c5a53f5b393e27b1f5319
-def gaussian_blur(img, kernel_size=3, sigma=5):
-    """
-    helper function for blurring the image, will soon be replaced by
-    tfa.image.gaussian_filter2d
-
-    :param img: the current state of the image
-    :param kernel_size: size of the convolution kernel used for blurring
-    :param sigma: gaussian blurring constant
-    :return: the altered image
-    """
-    def gauss_kernel(channels, kernel_size, sigma):
-        """
-        Calculates the gaussian convolution kernel for the blurring process
-
-        :param channels: amount of feature channels
-        :param kernel_size: size of the kernel
-        :param sigma: gaussian blurring constant
-        :return: the kernel for the given values
-        """
-        axis = tf.range(-kernel_size // 2 + 1.0, kernel_size // 2 + 1.0)
-        xvals, yvals = tf.meshgrid(axis, axis)
-        kernel = tf.exp(-(xvals ** 2 + yvals ** 2) / (2.0 * sigma ** 2))
-        kernel = kernel / tf.reduce_sum(kernel)
-        kernel = tf.tile(kernel[..., tf.newaxis], [1, 1, channels])
-        return kernel
-
-    gaussian_kernel = gauss_kernel(tf.shape(img)[-1], kernel_size, sigma)
-    gaussian_kernel = gaussian_kernel[..., tf.newaxis]
-
-    return tf.nn.depthwise_conv2d(img, gaussian_kernel, [1, 1, 1, 1],
-                                  padding='SAME', data_format='NHWC')
-
-
+#pylint: disable=R0914
```

Comment:
This should be removed. You can use up to 10 arguments per function. This function only takes 9, so removing should have no effect.